src/test/java/uk/gov/pay/ledger/transaction/model/CsvTransactionFactoryTest.java

```diff
@@ -403,6 +403,46 @@ void toMapShouldIncludeShouldInclude3DSecureRequired() {
         assertThat(csvDataMap.get("3-D Secure Required"), is(true));
     }
 
+    @Test
+    void toMapShouldReturnMapWithNullValueFor3DSecureRequiredWhenRequires3dsIsFalse() {
+
+        TransactionEntity transactionEntity = aTransactionFixture()
+                .withState(TransactionState.FAILED_REJECTED)
+                .withTransactionType(TransactionType.PAYMENT.name())
+                .withAmount(100L)
+                .withGatewayTransactionId("gateway-transaction-id")
+                .withCreatedDate(ZonedDateTime.parse("2018-03-12T16:25:01.123456Z"))
+                .withCorporateCardSurcharge(23)
+                .withCardBrandLabel("Visa")
+                .withDefaultCardDetails()
+                .withFee(5L)
+                .withTotalAmount(123L)
+                .withRequires3ds(false)
+                .withVersion3ds("2.0.1")
+                .withDefaultTransactionDetails()
+                .toEntity();
```

Comment:
Looks like this could be a bit less verbose by following the pattern in `toMapShouldIncludeShouldInclude3DSecureRequired()`.